docs/guides/hedgehog-engine/unleashed/levels/glvl.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Sonic GLvl setup tutorial for Sonic Unleashed
+---
+
+# How to use Sonic GLvl with Sonic Unleashed
+
+!!! note
+    This guide covers how to get unleashed levels working in order to edit them, this guide does not cover how to use GLvl. its recommended you also read the [Generations GLvl guide.](/guides/hedgehog-engine/blueblur/levels/glvl.md){ target="_blank"}
+
+
+    
+!!! info
+
+    This guide uses the following tools:
+
+    - [Sonic GLvl](/tools/hedgehog-engine/blueblur/levels/){ target="_blank"}
+    
+    - [Unleashed Mod Toolbox](/tools/hedgehog-engine/unleashed/files){ target="_blank"}  
+
+    - [Unleashed Object Template](assets/glvl/UnleashedObjectTemplate.zip)
+
+    - Xbox 360 SDK, find this yourself
+
+
+
+
+
+
+
+#### Part 1 (Preparing .ar Files)
+This example will be editing  Windmill Isle Act 1 Day. Open your extracted unleashed game folder and locate your files. In my case the files are #ActD_MykonosAct1.ar.00, #ActD_MykonosAct1.arl, ActD_MykonosAct1.ar.00, ActD_MykonosAct1.ar.01, ActD_MykonosAct1.arl.
```

Comment:
Maybe use bullet points or a code block to write the files instead of having them in a single line. I'll leave the choice to you, but having everything in a single line is just bad for reading